x/oracle/types/assets.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+//nolint

```

Comment:
```suggestion
// Denomination constants
```

Reply:
I have saved with the auto-format, as well as used gofmt -s , and go fmt -w, and still receive the error.